plugins/ocm/src/helpers/apiClient.ts

```diff
@@ -2,24 +2,22 @@ import { ConfigApi } from '@backstage/core-plugin-api';
 import { ErrorResponseBody } from '@backstage/errors';
 import { ClusterDetails } from '@janus-idp/backstage-plugin-ocm-common';
 
-const clusterApiFetchCall = (
-  configApi: ConfigApi,
-  params: string,
-): Promise<any> => {
+const clusterApiFetchCall = async (configApi: ConfigApi, params?: string) => {
   const backendUrl = configApi.getString('backend.baseUrl');
-  const jsonResponse = fetch(`${backendUrl}/api/ocm/status${params}`).then(r =>
-    r.json(),
+  const jsonResponse = await fetch(
+    `${backendUrl}/api/ocm/status${params || ''}`,
   );
-  return jsonResponse;
+  return jsonResponse.json();
```

Comment:
I think this can be put on a single line.

Reply:
It can but it would look cluttered.

```ts
return (await fetch(`${backendUrl}/api/ocm/status${params || ''}`)).json()
```

Reply:
Yea I guess the original way is more readable.